Wikipedia/UI-V5/WMFArticleContainerViewController.m

```diff
@@ -557,6 +576,71 @@ - (void)sectionEditorFinishedEditing:(SectionEditorViewController*)sectionEditor
     [self fetchArticle];
 }
 
+
+
+
+
+
+
+
+
+
+
+
+#pragma mark - UIViewControllerPreviewingDelegate
+
+- (nullable UIViewController*)previewingContext:(id<UIViewControllerPreviewing>)previewingContext
+                      viewControllerForLocation:(CGPoint)location {
+
+//return
+//[[WMFArticleContainerViewController alloc] initWithArticleTitle:[[MWKTitle alloc] initWithURL:[NSURL URLWithString:@"http://en.wikipedia.org/wiki/Cat"]]
```

Comment:
Try disabling all our JS stuff that does touch handling (i.e. return early from `www/listeners.js#maybeSendMessageForTarget`) to see if that makes a difference. If so, we might need to make some more invasive changes w/ how we intercept navigation interactions...